android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -34,9 +35,16 @@ public MusicControlNotification(MusicControlModule module, ReactApplicationConte
         String packageName = context.getPackageName();
 
         // Optional custom icon with fallback to the play icon
-        smallIcon = r.getIdentifier("music-control-icon", "drawable", packageName);
+        smallIcon = r.getIdentifier("music_control_icon", "drawable", packageName);
         if(smallIcon == 0) smallIcon = r.getIdentifier("play", "drawable", packageName);
     }
+    
+    public void setCustomNotificationIcon(String resourceName) {
+      Resources r = this.context.getResources();
+      String packageName = this.context.getPackageName();
+
+      customIcon = r.getIdentifier(resourceName, "drawable", packageName);
```

Comment:
This is the last thing: you're not following the same code style, using two spaces instead of four.
Also, you can remove `this` from the context, it's not really needed

Other than that, it looks good :+1: